gobblin-service/src/main/java/org/apache/gobblin/service/monitoring/DagActionStoreChangeMonitor.java

```diff
@@ -130,30 +132,34 @@ protected void processMessage(DecodeableKafkaRecord message) {
     String changeIdentifier = tid + key;
     if (!ChangeMonitorUtils.shouldProcessMessage(changeIdentifier, dagActionsSeenCache, operation,
         produceTimestamp.toString())) {
+      this.messageFilteredOutMeter.mark();
       return;
     }
 
+    // Used to easily log information to identify the dag action
+    DagActionStore.DagAction dagAction = new DagActionStore.DagAction(flowGroup, flowName, flowExecutionId,
+        dagActionType);
+
     // We only expect INSERT and DELETE operations done to this table. INSERTs correspond to any type of
     // {@link DagActionStore.FlowActionType} flow requests that have to be processed. DELETEs require no action.
     try {
       if (operation.equals("INSERT")) {
         if (dagActionType.equals(DagActionStore.FlowActionType.RESUME)) {
-          log.info("Received insert dag action and about to send resume flow request");
+          log.info("Received insert dag action and about to send resume flow request for: {}", dagAction);
```

Comment:
nit: too conversational.  how about:
```
log.info("DagAction change ({}): {}", operation, dagAction)
```
(i.e. won't the `resume/kill/launch` be logged as `dagAction.dagActionType`)?

Reply:
I was thinking about unifying all the logs like this as well, let me make the change. I will add a bit more context in message but use this format.